3.7/s2i/bin/usage

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+DISTRO=`cat /etc/*-release | grep ^ID= | grep -Po '".*?"' | tr -d '"'`
+NAMESPACE=centos
+[[ $DISTRO =~ rhel* ]] && NAMESPACE=rhscl
+
+cat <<EOF
+This is a S2I python-3.6 ${DISTRO} base image:
```

Comment:
Isn't 3.6 here just a forgotten `/3.6/3.7/`?

Reply:
I think it is caused by not using `distgen` generator. Please, refer to https://github.com/sclorg/s2i-python-container#contributing for more info. Let's take this as a UX experiment, If the description there does not explain enough what is necessary, we need to improve the documentation and make it easier to contribute.

Reply:
Yes, you are right. I thought that distgen is used automatically when you do `make build` and I wasn't aware of `multispec.yml` file so I did all changes manually. My fault.

Is there any documentation of `multispec.yml`? Because I want to add Fedora 29 with Python 3.7 which is a new combination of distro and Python version and some options in `multispec.yml` aren't self-describing. For example, what does `python_img_version: "1"` mean?